styles/style.css

```diff
@@ -81,20 +89,146 @@ textarea {
     appearance: none;
     border-radius: 0;
 }
+
 .btn {
     display: flex;
     justify-content: center;
-	align-items: center;
+    align-items: center;
     padding: 8px;
     width: 100%;
     max-width: 94px;
     border-radius: 2px;
     color: var(--color-white);
     font: var(--font-button-default);
 }
+
 .btn.btn--primary {
     padding: 16px;
     max-width: 163px;
     background: var(--color-accent);
     font: var(--font-button-small);
 }
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.container {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1177px;
+}
+
+.site-header .header-top {
+    padding: 14px 0;
+    background-color: #7E33E0;
+}
+.site-header .header-top a {
+    color: var(--color-white);
+    line-height: 18.75px;
+    font-size: 16px;
+    font-family: Arial, "Josefin Sans", sans-serif;
```

Comment:
font-weight doesn't match the layout in figma
<img width="499" alt="image" src="https://github.com/user-attachments/assets/8454722a-cd99-4e56-ae91-5395320c184a">

Reply:
corrected